doc/source/user_guide/ug_implementation.rst

```diff
@@ -174,13 +174,15 @@ variable ``dumpfreq``. The namelist variable ``ice_ic`` contains the
 pointer to the filename from which the restart data is to be read and 
 the namelist option ``restart`` must be set to ``.true.`` to use the file.
 ``dump_last`` namelist can also be set to true to trigger restarts automatically
-at then end of runs.
+at then end of runs. The default restart file format is binary and is set
```

Comment:
change 'then' to 'the'.
Also, the 'set by setting' language is a little weird.  How about 

> The default restart file format is binary, set in namelist as ``restart_format = 'bin'``. For netCDF, set ``restart_format = 'nc'``.

?